.travis.yml

```diff
@@ -1,3 +1,3 @@
 language: node_js
 node_js:
-  - "0.10"
\ No newline at end of file
+  - "0.12"
```

Comment:
The issue was for-of loop that used Symbols which doesn't work with Node 10.XX. and 11.XX. Updated code to use forEach loop instead of for-of loop to solve the issue.